src/index.js

```diff
@@ -302,10 +302,34 @@ const addLang = (tree, {rules: {lang}}) => {
   return tree;
 };
 
+const commentsFormatting = tree => {
+  tree.walk = walk;
+  tree.walk(node => {
+    if (typeof node === 'string' && /<!--[^]*-->/gm.test(node.trim())) {
+      const originalComments = node.trim();
```

Comment:
`trim` line: 308,309 - can be done once at the very beginning of the condition

Reply:
yea make sense

Reply:
`node` can be object as well, so it will show `node.trim() not a funtion` if we do not check that type `string`. 
So, I guess we need to keep like this only. 

What do you think ?

Reply:
I thought to do something like this before the condition 
```js
node = node?.trim()
```